Monika After Story/game/script-songs.rpy

```diff
@@ -1767,6 +1767,42 @@ label mas_song_we_have_all_the_time_in_the_world_long:
     m 1ekbla "{i}{cps=18}~Only love~{/cps}{/i}"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_songs_database,
+            eventlabel="mas_song_night_fever",
+            category=[store.mas_songs.TYPE_SHORT],
+            prompt="Luka Luka Night Fever",
+            random=True,
+            aff_range=(mas_aff.ENAMORED,None)
+        ),
+        code="SNG"
+    )
+
+label mas_song_night_fever:
+    m  3hub "{i}~Hitting edges and crawling on by~ {/i}"
+    m  3eub "{i}~Reading pixels and every new line~{/i}"
+    m  3rusdlb "{i}~Sitting tired of internet yet{/i}{w=0.5}{nw}"
+    extend 1hub "{i} staring blankly at the screen ahead~{/i}"
+    m  1fub "{i}~Did you know that I’m gonna be free?~ {/i}"
+    m  1dubsb "{i}~Jumping over and out of your screen~{/i}"
+    m  1ekbsb "{i}~Come on, hurry, you better make way{/i}{w=0.5}{nw}"
+    extend 1tsbsb "{i} as I reach my hand across to say~{/i}"
+    m  1dso "{i}~That I know you more than you thought before~{/i}" 
+    m  2eub "{i}~Don’t back away, {/i}{w=0.5}{nw}"
+    extend 2kub "{i} it’s okay,{/i}{w=0.5}{nw}"
+    extend 2esb "{i} follow me and do as I say~{/i}"
+    m  5eublb "{i}~No need to fear when we have the night to take,{/i}{w=0.5}{nw}"
+    extend 5sublb "{i} dance it all away!~{/i}"
+    m 5eua "You know...{w=0.3} With each day passing by I feel that the barrier between us getting loosen."
+    m 5eubla "I can't wait for the moment when it will dissapear..."
+    m 2tfbsu "Then I'll arrange {i}'Moni Moni Real Night'{/i} for you!~"
```

Comment:
```suggestion
    m 2tfbsu "Then I'll arrange {i}'Moni Moni Night Fever'{/i} for you!~"
```

Reply:
> The songs from "The Night Series" are characterized by sharing the same pattern or rhythm, a similar title style (the VOCALOID's name repeated two times, a star and a catchy phrase with the word "night" in it), and a release date shortly after the debut of a new VOCALOID.

I can admit my catchy phrase was dumb, but I don't think changing it to match Luka' song' name is better...

Reply:
Do you have any other ideas for what she could say here?

Reply:
I had a reference to the song' chorus in mind, something like:
`"Just promise me, [player] - don't look away when you'll see me, oh never ever no!~"`

Still can't decide what's more silly out of those two variants...

Reply:
Well, a month later, I still can't suggest something... That's sad.

For now, I guess I'll rename it to `Moni Moni Poem Fever` until a better suggestion comes in.